cdn/dovetail-cdn.yml

```diff
@@ -13,53 +13,105 @@ Metadata:
       - Label:
           default: Distribution Parameters
         Parameters:
-          - OriginBucket
           - DistributionDomain
           - ExpiredRedirectPrefix
           - OriginRequestFunctionArn
-          - EnvironmentTypeAbbreviation
+          - EnvironmentType
+      - Label:
+          default: Primary Region
+        Parameters:
+          - CacheBehaviorPrefix
+          - OriginBucket
           - RealtimeLogKinesisStreamArn
+      - Label:
+          default: Secondary Region (optional)
+        Parameters:
+          - CacheBehaviorPrefix2
+          - OriginBucket2
+          - RealtimeLogKinesisStreamArn2
+      - Label:
+          default: Tertiary Region (optional)
+        Parameters:
+          - CacheBehaviorPrefix3
+          - OriginBucket3
+          - RealtimeLogKinesisStreamArn3
     ParameterLabels:
       OriginBucket:
-        default: Primary dovetail-cdn-arranger S3 Bucket
+        default: Primary region's dovetail-cdn-arranger S3 Bucket
+      OriginBucket2:
+        default: Secondary region's dovetail-cdn-arranger S3 Bucket
+      OriginBucket3:
+        default: Tertiary region's dovetail-cdn-arranger S3 Bucket
       DistributionDomain:
         default: Domain Name
       ExpiredRedirectPrefix:
         default: Prefix for redirecting back to Dovetail
       OriginRequestFunctionArn:
         default: Dovetail origin request lambda
-      EnvironmentTypeAbbreviation:
-        default: Environment type used in naming things
+      EnvironmentType:
+        default: Environment Type
+      CacheBehaviorPrefix:
+        default: Primary region's path prefix
+      CacheBehaviorPrefix2:
+        default: Secondary region's path prefix
+      CacheBehaviorPrefix3:
+        default: Tertiary region's path prefix
       RealtimeLogKinesisStreamArn:
-        default: Kinesis stream for realtime logs
+        default: Primary region's kinesis stream for realtime logs
+      RealtimeLogKinesisStreamArn2:
+        default: Secondary region's kinesis stream for realtime logs
+      RealtimeLogKinesisStreamArn3:
+        default: Tertiary region's kinesis stream for realtime logs
 
 Parameters:
   OriginBucket:
     Type: String
     Description: eg. some-bucket-name.s3.amazonaws.com
+  OriginBucket2:
+    Type: String
+    Description: eg. some-bucket-name.s3.amazonaws.com
```

Comment:
Should [prefer](https://github.com/PRX/internal/wiki/AWS:-Best-Practices#amazon-s3) `[mybucket].s3.[region].amazonaws.com`

Reply:
Ah, right.  Non-us-east-1 buckets!